src/components/ChatBot/ChatFloatingModal.tsx

```diff
@@ -77,7 +77,7 @@ export default function ChatFloatingModal () {
 	};
 
 	const { network } = useNetworkContext();
-	const grillData = {
+	const grillData: { [index: string]: string[] } ={
```

Comment:
you can take grillData outside the main component (ChatFloatingModal) because it's not dependent on any variable and also it's a const so it will be never changed,